packages/cashier/src/components/cashier-onboarding/cashier-onboarding.tsx

```diff
@@ -163,7 +164,7 @@ const CashierOnboarding = observer(({ setSideNotes }: TCashierOnboardingProps) =
             options.push(Providers.createPaymentAgentProvider(onClickPaymentAgent));
         }
 
-        if (show_p2p_in_cashier_onboarding) {
+        if (is_p2p_enabled || (has_fiat_currency && has_usd_currency)) {
```

Comment:
```suggestion
        if (is_p2p_enabled || has_usd_currency) {
```
If I'm not mistaken `has_usd_currency` already contains `has_fiat_currency` logic in it, So we can omit `has_fiat_currency`? 🤔

Reply:
true, i think we can remove the has_fiat_currency and just check using has_usd_currency as p2p users are only allowed if they are using USD anyways 👍